modules/cnf-about-ptp-fast-event-notifications-framework.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-about-ptp-fast-event-notifications-framework_{context}"]
+= About the PTP fast event notifications framework
+
+DU applications can subscribe to PTP fast events notifications that are generated by {product-title} with the PTP Operator and `cloud-event-proxy` sidecar container.  You enable the `cloud-event-proxy` sidecar container by setting the `enableEventPublisher` field to `true` in the `ptpOperatorConfig` custom resource (CR) and specifying a `transportHost` address. PTP fast events use an AMQP event notification bus that is provided by the AMQ Interconnect Operator. AMQ Interconnect is based on Dispatch Router from the Apache Qpid™ project.
+
+The `cloud-event-proxy` sidecar container can access the same resources as the primary vRAN application without consuming any of the resources of the primary application. In addition, there is no significant latency when communicating between the primary and sidecar application.
+
+The fast events notifications framework uses a REST API for communication and is based on the O-RAN REST API specification. The framework consists of a publisher, subscriber, and an AMQ messaging bus to handle communications between the publisher and subscriber applications. The `cloud-event-proxy` sidecar is a utility container that runs in a pod that is loosely coupled to the main application container on the DU node. It provides an event publishing framework that allows you to subscribe DU applications to published PTP events.
+
+The following is a a high level overview of the PTP fast events consumer workflow for DUs:
+
+DU application requests a subscription:: The DU sends an API request to the `cloud-event-proxy` sidecar to create a PTP events subscription. The `cloud-event-proxy` sidecar creates a new subscription resource.
```

Comment:
DU Application have to run cloud-event-proxy container   in sidecar pattern, to subscribe for PTP events